daffodil-lib/src/main/scala/org/apache/daffodil/util/RangeBound.scala

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
So I see you pulled this code out of the file below in order to share it, which is great. 

I'm going to assume this didn't change in some subtle way as well. If there's something to look at for delta review in this that's in some way subtle, please highlight for me. Otherwise I'm assuming this code is ok.

Reply:
Sorry, I should have highlighted it when I put up the PR. It should have a comment now. Bassically just a Range type-alias.